extensions/@shopgate-theme-config/frontend/config-common.js

```diff
@@ -43,6 +43,59 @@ export default {
             name: 'ShopgateProductOrderQuantityHint',
             id: '@shopgate/engage/product/OrderQuantityHint',
           },
+          {
+            name: 'ShopgateProductSwatch',
+            id: '@shopgate/engage/product/Swatch',
+            settings: {
+              selectionStyles: {
```

Comment:
Those are just styles. Please move it to `styles`

Reply:
Actually it's an object with sub-objects containing styles. I've put it here because it's not just styles. Maybe there is another way, though.